GiniVision/Classes/GiniVision.swift

```diff
@@ -17,30 +17,40 @@ import UIKit
 @objc public protocol GiniVisionDelegate {
     
     /**
+     Called when the user has taken a picture or imported a file (image or PDF) from camera roll or document explorer
```

Comment:
If the user imported a file... it's not capturing anymore, is it? I guess, the name `didCapture` is misleading.

Reply:
I thought about that, but I wasn't sure that `didPick` was a better option. What do you think?

Reply:
`didImportImage`?
`didChooseImage`?

Reply:
This is also called when a image is captured, so they are not correct.